jsapp/js/libs/reactCreateBemElement.d.ts

```diff
@@ -0,0 +1,13 @@
+interface BemComponent extends React.ComponentClass {
+  __: (className: string, elementType?: string) => BemComponent
+}
+
+export interface BemDefinition {
+  [elementName: string]: BemComponent
+  create: (className: string, elementType?: string) => BemComponent
+}
+
+export const BEM: {
+  (className: string, elementType?: string): BemComponent
```

Comment:
I'm a little confused as why this is here :P Does the `()` syntax mean it's expecting the `BemDefinition` to contain `BemComponents`?

Reply:
This weird syntax means that `BEM` is a function :-D

Reply:
I could've wrote just `type BEM = (className: string, elementType?: string) => BemComponent` but unfortunately it's a weird code here and this function is both a function and an object with `initi` function property… 😵‍💫

Reply:
Got it!